easybuild/main.py

```diff
@@ -199,6 +200,10 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
     eb_go, cfg_settings = set_up_configuration(args=args, logfile=logfile, testing=testing)
     options, orig_paths = eb_go.options, eb_go.args
 
+    if options.include_easyblocks_from_pr:
+        included_easyblocks = fetch_easyblocks_from_pr(options.include_easyblocks_from_pr)
+        include_easyblocks(options.tmpdir, included_easyblocks)
```

Comment:
not sure if this is the correct `tmpdir`

Reply:
You better create a subdirectory in `self.tmpdir`, to avoid that `--include-easyblocks` and `--include-easyblocks-from-pr` step on each others toes if both options are used at the same time.

Or we tweak the `include_easyblocks` function to use `tempfile.mkdtemp` to create a subdirectory in the path that it is given, that's probably a better options in the long run:

```python
easyblocks_path = tempfile.mkdtemp(prefix=tmpdir, os.path.join(dir=tmpdir, prefix='included-easyblocks-')
```

Ideally, we also have this use case covered of using both `--include-easyblocks*` options in the tests...

Reply:
hm, when looking into adding a test for the use of both `--include-easyblocks*` options, I noticed that `--list-easyblocks` doesn't take into account the easyblocks from a PR, because it's done in the `options.postprocess` step, before `set_up_configuration`...